City-of-Helsinki/drupal-asuntotuotanto
# SATT-117: Undefined variable  IndexParamBuilder.php

### Description

Fixing Elastic index [warning] Undefined variable $value IndexParamBuilder.php


### How to test it

- make fresh
- make shell
- drush sapi-s
- drush sapi-i apartment
   - check that there isn't Undefined variable $value IndexParamBuilder
